kafka-connect-s3/src/test/java/io/confluent/connect/s3/DataWriterByteArrayTest.java

```diff
@@ -87,6 +88,23 @@ public void tearDown() throws Exception {
 
   @Test
   public void testNoSchema() throws Exception {
+    CompressionType compressionType = CompressionType.GZIP;
+    localProps.put(S3SinkConnectorConfig.FORMAT_CLASS_CONFIG, ByteArrayFormat.class.getName());
+    localProps.put(S3SinkConnectorConfig.COMPRESSION_TYPE_CONFIG, compressionType.name);
+    setUp();
+    task = new S3SinkTask(connectorConfig, context, storage, partitioner, format, SYSTEM_TIME);
+
+    List<SinkRecord> sinkRecords = createByteArrayRecordsWithoutSchema(7 * context.assignment().size(), 0, context.assignment());
+    task.put(sinkRecords);
+    task.close(context.assignment());
+    task.stop();
+
+    long[] validOffsets = {0, 3, 6};
+    verify(sinkRecords, validOffsets, context.assignment(), ".bin", compressionType);
+  }
+
+  @Test
+  public void testGzipCompression() throws Exception {
```

Comment:
mismatch of test method names. Lets keep the old method first and then add `testGzipCompression` below it.